components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2590,20 +2590,11 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
             if (!host) {
                 throw Error("Unknown host: " + parsedUrl?.host);
             }
-            if (host === "github.com" && this.config.githubApp?.enabled) {
```

Comment:
I believe we want to remove this but could use a second opinion - @AlexTugarev

Reply:
Let's examine the effect of this change: 
1. on Cloud: the subsequently created projects for github.com repositories will require to create webhooks if you check `Enable Prebuilds` in settings ✔️ 
    ➡️ we need to make sure, there are no other checks of this kind that would prevent installation of webhooks. (could be in ScmService.installWebhook)

Reply:
I was thinking about this change https://github.com/gitpod-io/gitpod/commit/936302b6a632293233dd72e02fbf8d5a31e89fdb.